rook/main.py

```diff
@@ -45,28 +47,90 @@
 
 args = parser.parse_args()
 
-if args.load_average > 0 and hasattr(os, "getloadavg"):
-  #Note that this is also done before starting each test
-  while os.getloadavg()[0] > args.load_average:
-    print("Load Average too high, waiting ", os.getloadavg()[0])
-    time.sleep(1.0)
+if socket.gethostname() == "VDI-RavenW7-01":
+  print("Overridding load average")
+  args.load_average = 4.0
 
-def load_average_adapter(function):
-  """
-    Adapts function to not start until load average is low enough
-    @ In, function, function, function to call
-    @ Out, new_func, function, function that checks load average before running
-  """
-  def new_func(data):
+if hasattr(os, "getloadavg"):
+  def load_average_adapter(function):
+    """
+      Adapts function to not start until load average is low enough
+      @ In, function, function, function to call
+      @ Out, new_func, function, function that checks load average before running
+    """
+    def new_func(data):
+      """
+        function that waits until load average is lower.
+        @ In, data, Any, data to pass to function
+        @ Out, result, result of running function on data
+      """
+      while os.getloadavg()[0] > args.load_average:
+        time.sleep(1.0)
+      return function(data)
+    return new_func
+
+else:
+  import psutil
+
+  class LoadClass(threading.Thread):
```

Comment:
Do you really need to define a class/function in if statements? Can't it be defined at the module level? For pickiling (in case we need it), this will create problems

Reply:
The alternative to defining LoadClass in an ```if``` statement is to make psutil a requirement on Linux and Mac, instead of just on Windows.  I can change that if you want.

Reply:
I created an alternative pull request #898  that does this in a different way.